src/main/java/org/zalando/nakadi/service/timeline/TimelineService.java

```diff
@@ -184,6 +188,7 @@ private void switchTimelines(final Timeline activeTimeline, final Timeline nextT
                     final Timeline.StoragePosition storagePosition =
                             topicRepositoryHolder.createStoragePosition(activeTimeline);
                     activeTimeline.setLatestPosition(storagePosition);
+                    scheduleTimelineCleanup(activeTimeline);
```

Comment:
Such a precision will not make much sense here as the Job that cleans timelines is run once per hour.